src/mpyl/build.py

```diff
@@ -110,7 +110,7 @@ def run_mpyl(
                     properties=run_properties,
                     steps_collection=StepsCollection(logger=logger),
                 )
-                run_result = run_build(run_plan, steps, reporter, cli_parameters.local)
+                run_result = run_build(run_plan, steps, reporter, cli_parameters.dryrun)
```

Comment:
maybe this should be `cli_parameters.local or cli_parameters.dryrun` @koeves ?

Reply:
agreed

Reply:
If I change it back to `cli_parameters.local` dryrun doesn't work. The `run_build` function has dryrun as its third parameter. Should I add an extra parameter for this?